tests/unit/Trace/Reporter/ZipkinReporterTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenCensus\Tests\Unit\Trace\Reporter;
+
+use OpenCensus\Trace\Reporter\ZipkinReporter;
+use OpenCensus\Trace\TraceContext;
+use OpenCensus\Trace\TraceSpan;
+use OpenCensus\Trace\Tracer\TracerInterface;
+use Prophecy\Argument;
+
+/**
+ * @group trace
```

Comment:
For now, they are all in the trace group. OpenCensus does eventually plan to have other metrics and stats reporting in the future.